### DISCUSSION TITLE
No me deja terminar el juego se queda parado en la cinemática final

### DISCUSSION THREAD

**Discussion by eliascejas** [mupen64plus-ae/mupen64plus-ae#996] [Q&A]
Cómo lo arreglo

**Comment by fzurita** [comment]
Cual juego?

Puedes hacer un vídeo?

**Reply by eliascejas** [reply]
![Screenshot_20220125-125934](https://user-images.githubusercontent.com/98409042/151012414-5e9b128b-90e8-4f4c-beda-17ebc25a7208.png)

**Reply by eliascejas** [reply]
No me deja terminar se queda trabado

**Comment by eliascejas** [comment]
Zelda ocarina of time

**Comment by fzurita** [comment]
Estás ocupando un hack o traducción del juego?

**Reply by fzurita** [reply]
Ok, veo que estás ocupando una traducción. Es un problema de que ya se sabe. Lo que tienes que hacer es ocupar la opción de iniciar en vez de reanudar.

**Comment by eliascejas** [comment]
Sí estoy usando una
 traducción al español

**Comment by eliascejas** [comment]
Muchas gracias

**Comment by eliascejas** [comment]
Hola perdón por molestar pero me sigue sin funciona

**Reply by fzurita** [reply]
O sea terminaste el juego de nuevo y se congelo ahí mismo?

Si buscas por videos en Youtube verás que no pasa nada después y hay que resetear la consola para comenzar el juego de nuevo

**Comment by eliascejas** [comment]
Sí ahí mismo no me deja continuar

**Comment by Sus2824** [comment]
Ayuda, mi juego se quedo en blanco 
![Screenshot_20240512_135200](https://github.com/mupen64plus-ae/mupen64plus-ae/assets/169614630/f75cea9a-b3f2-4505-a56c-e6d4f94de664)
Cuando paso el castillo muerto de zelda master of time se queda asi y se escucha, pero se quesa en blanco, me costo mucho llegasr hasta ahi😭

**Reply by Sus2824** [reply]
![Screenshot_20240512_140040](https://github.com/mupen64plus-ae/mupen64plus-ae/assets/169614630/c451311f-8b53-46d3-b381-23d03ce468a6)
Si entro ahi se queda en blanco, se congela
